src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -658,6 +667,28 @@ tIndividualFontSettings P036_data_struct::CalculateIndividualFontSettings(uint8_
       LineSettings[k].ypos = LineSettings[k - 1].ypos + FontSizes[LineSettings[k - 1].fontIdx].Height + lSpace;
     }
   }
+# ifdef P036_CHECK_INDIVIDUAL_FONT
+
+  if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+    String log1;
+
+    if (log1.reserve(140)) { // estimated
+      delay(10);             // otherwise it is may be to fast for the serial monitor
+      log1.clear();
```

Comment:
- `delay()` calls should be _really_ required, or else limited to 1 msec,, and only to avoid the watchdog stepping in (after 30 seconds)
- no need to `clear()` a freshly declared String, that's already empty...